code/modules/mob/living/simple_animal/revenant.dm

```diff
@@ -179,7 +179,7 @@
 	if(sanitize)
 		message = trim(copytext_char(sanitize(message), 1, MAX_MESSAGE_LEN))
 	src.log_talk(message, LOG_SAY)
-	var/rendered = span_revennotice("<b>[src]</b> says, \"[message]\"")
+	var/rendered = span_deadsay("<b>[src]</b> says, \"[message]\"")
```

Comment:
maybe it would be best to give them some unique color for deadchat talking, or bolding or somethin